index.csv

```diff
@@ -225,3 +225,6 @@ longest,MIT,1.0.1
 repeat-element,MIT,1.1.2
 trim,MIT,0.0.1
 css-value,MIT,0.0.1
+rework,MIT,1.0.1
+process,MIT,0.5.2
```

Comment:
this also applies to v0.6.0, v0.7.0, v0.8.0, v0.9.0, v0.10.0, v0.10.1, v0.11.0, and v0.11.1 - can you add those entries as well?